### ISSUE TITLE
Active works on LFE.

### ISSUE THREAD

**Issue by prertik** [lfe/docs#123]
We know the world domination of Lisp is very slow and the curse of the Lisp has surely caught up with LFE as very few people even in the Erlang spec seem to know of LFE. I felt there was a need to let people know that there is Lisp even for Erlang so I wrote about it in [learnxinyminutes.com](https://learnxinyminutes.com/docs/lfe/). While trying to use some perfect LFE programs as examples to show Erlang is cool, I couldn't find much references but there were some in Wikipedia which were better options so I used it. @oubiwann did an amazing work with starting lfe/docs which would really be impossible without his contributions. Like @oubiwann said, lfe is yak hair. Nothings better than lisp. I tried different languages but felt home in Lisp. I used Elixir for some time, hated Erlang due to prolog-y syntax but when I found out about lfe via [http://lfe.io/](http://lfe.io/). I knew this was it. LFE for Erlang, Clojure for Java, Chicken Scheme for low level. Lisp is really everywhere.

Now, rather than rant about the missing things or creating lists of LFE lacks. Let's make LFE great. We need an active maintainer who can review pull requests and help with issues not longer than 2 days old. If given a proper guidance and proper mentor, I will write all the documentations for LFE. Then we can work on other issues like making property based testing and many more.

**Let's make LFE a complete language with best documentations and best tooling support. 
Let's revive Lisp.**
>Let's start Revenge of the Lisp.

**Comment by oubiwann** [comment]
Love your enthusiasm for Lisp in general and LFE in particular :-)

Before introducing new processes, etc., let's use what's already available to us and play it by ear, taking each PR as a step in a better direction for LFE docs.

There are a bunch of open tickets for the LFE docs, as mentioned when we talked here:
 * https://github.com/lfe/docs/issues/119#issuecomment-322103446

More details are available on the LFE mail list at the following links:
 * https://goo.gl/XWMyvy - LFE Docs Update
 * https://goo.gl/ZKQtZE - More updates about the docs and project management / organization

In short, start picking tickets, asking for help as necessary, submitting PRs, and moving things forward :-) 

Ray of hope:
 * As soon as we get enough updates to populate the links that need to be added to this page:  http://docs.lfe.io/dev/, we can make a new docs release with the new style 
 * At that point, /current will point to the latest release (with the new style), and /dev will continue to point to the ongoing docs development
 * The old docs will continue to be available at the old version number

**Comment by oubiwann** [comment]
Lots has changed since my follow-up comment ... closing
